src/zsl/interface/gearman/task_filler.py

```diff
@@ -4,12 +4,11 @@
 
 .. moduleauthor:: Martin Babka
 """
-from __future__ import print_function, unicode_literals
+from __future__ import print_function, unicode_literals, absolute_import
```

Comment:
while this is required? i do not see any usage of `absolute_import` in this file.

Reply:
see, it's future :) :+1: but why was this change necessary?